protos/follow_me/follow_me.proto

```diff
@@ -30,19 +30,42 @@ service FollowMeService {
 
 // Configuration type.
 message Config {
-    // Direction relative to the target that the vehicle should follow
+    /** [!DEPRECIATED!]
+      * Direction relative to the target that the vehicle should follow.
+      * Measured in reference to target's course (where it's moving towards)
+      * 
+      * It is only used for setting the follow angle indirectly, for legacy reasons
+      * (since PX4 Follow Me used to use FollowDirection enums for setting, but now
+      * uses the follow_angle_deg float value).
+      * 
+      * DO NOT USE if possible!
```

Comment:
@junwoo091400 so what's the new API? Just an angle to define this? And if so, would it make sense to through this API out and have the new one with MAVSDK v2 which is probably going to be released in the next weeks anyway?

Reply:
The new API is to just setting the angle in degrees via the `FLW_TGT_FA` parameter. If removing this `follow_direction` is ok, I would love to! (Since it requires having switch cases, etc (Having enums is painful :rofl:)

Do I just drop it in this PR then?

Reply:
Yeah, just drop it :+1:.